pkg/controller.v1beta2/tensorflow/controller.go

```diff
@@ -406,6 +456,22 @@ func (tc *TFController) reconcileTFJobs(tfjob *tfv1beta2.TFJob) error {
 	return tc.updateStatusHandler(tfjob)
 }
 
+func getTotalReplicas(tfjob *tfv1beta2.TFJob) int32 {
```

Comment:
These should be in https://github.com/kubeflow/tf-operator/blob/master/pkg/controller.v1beta2/tensorflow/job.go